esmvaltool/recipes/hydrology/recipe_marrmot.yml

```diff
@@ -16,19 +16,16 @@ preprocessors:
   daily: &daily
     extract_shape:
       # Lumped model: needs catchment-aggregated input data
-      shapefile: meuse_hydrosheds.shp  # In aux (config-user.yml)
+      shapefile: Meuse/meuse.shp  # In aux (config-user.yml)
```

Comment:
Note that with the latest version we're using, I think we now use a capital for Meuse.shp